build_tools/ci/cpu_comparison/run_test.py

```diff
@@ -3,20 +3,17 @@
 # Copyright 2024 The IREE Authors
 
 import argparse
+from input_generator import generate_inputs, verify_determinism, load_input
+from matmul_template.matmul_generator import generate_matmul_test
+import numpy as np
 import os
+from pathlib import Path
 import re
+from output_comparer import compare
 import subprocess
+from textwrap import dedent
 import time
 import urllib.request
```

Comment:
the reason these keep getting shuffled is because i have actually two formetters - black and one that clears out dead `import`s and alphabetizes. i'd prefer to keep them alpha but if we're not going all jump on board i'll stop running the second formetter (or at least reset these lines).

Reply:
My motivation for this was the duplicated imports (7 red lines, 6 green lines) the ordering was an after though, by all means run that formatter whenever you like